Frontend/android-native/mobility-customer/src/main/java/in/juspay/mobility/customer/DatabaseHelper.java

```diff
@@ -0,0 +1,63 @@
+package in.juspay.mobility.customer;
+
+import android.content.Context;
+import android.content.ContentValues;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+import android.util.Log;
+
+import in.juspay.mobility.common.MobilityCommonBridge;
+import android.database.Cursor;
+import org.json.JSONObject;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DatabaseHelper extends SQLiteOpenHelper {
+
+    public static final String DATABASE_NAME = "DriverInfoDb";
+    public static final String DRIVERINFO_TABLE_NAME = "DriverInfo";
+    private static SQLiteDatabase db;
+    public static final String[] DRIVERINFO_COLUMNS = {"otp", "driverName", "vehicleDetails", "registrationNumber", "rating", "source", "destination", "price", "distance", "estimatedDistance", "driverNumber", "merchantExoPhone", "vehicleVariant"};
+
+
+    public DatabaseHelper(Context context) {
+      super(context,DATABASE_NAME,null,1);
+    }
+
+
+    public void deleteAllRows(SQLiteDatabase db){
+      db.delete(DRIVERINFO_TABLE_NAME, null, null);
+    }
+
+    public JSONObject getRecord(SQLiteDatabase db){
+      JSONObject data = new JSONObject();
+      try{
+          Cursor cursor = db.query(DRIVERINFO_TABLE_NAME, DRIVERINFO_COLUMNS, null, null, null, null, null);
```

Comment:
Can you make it generic ?  Add support for all the CRUD operations can be done from purescript itself.